ProductsService/ProductsService.Application/Validators/Categories/AddCategoryRequestValidator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Http;
+using ProductsService.Application.UseCases.CategoryUseCases.Commands.AddCategory;
+
+namespace ProductsService.Application.Validators.Categories;
+
+public class AddCategoryRequestValidator : AbstractValidator<AddCategoryRequest>
+{
+    public AddCategoryRequestValidator()
+    {
+        RuleFor(req => req.Name)
+            .NotEmpty()
+            .WithMessage("Wrong category name");
+
+        RuleFor(req => req.Image)
+            .Must(BeAnImage)
+            .WithMessage("You should upload only image files")
+            .When(req => req.Image is not null);
+    }
+
+    private bool BeAnImage(IFormFile? file)
```

Comment:
maybe better to name it IsAnImage?

Reply:
It's used as parameter of FluentValidation Must method, so i thought it's better to call it BeAnImage, to get something like Must(BeAnImage) when i call it